File: examples/petstore-broker/src/test/java/oas/example/petstore/broker/cucumber/context/TestContext.java
package oas.example.petstore.broker.cucumber.context;

import io.restassured.response.Response;

public class TestContext {
    private Response response;
    public Response getResponse() {
        return response;
    }

    public void setResponse(Response response) {
        this.response = response;
    }

}

File: app/spring/oas-stub-spring/src/test/kotlin/io/github/ktakashi/oas/cucumber/context/TestContext.kt
package io.github.ktakashi.oas.cucumber.context

import io.restassured.http.Header
import io.restassured.response.Response

data class TestContext(var applicationUrl: String,
                       var prefix: String,
                       var apiDefinitionPath: String = "",
                       var apiName: String = "",
                       var headers: MutableList<Header> = mutableListOf(),
                       var response: Response? = null,
                       var responseTime: Long? = null)
